authenticatorbridge/src/main/java/com/bitwarden/authenticatorbridge/factory/AuthenticatorBridgeFactory.kt

```diff
@@ -0,0 +1,33 @@
+package com.bitwarden.authenticatorbridge.factory
+
+import android.content.Context
+import com.bitwarden.authenticatorbridge.manager.AuthenticatorBridgeManager
+import com.bitwarden.authenticatorbridge.manager.AuthenticatorBridgeManagerImpl
+import com.bitwarden.authenticatorbridge.manager.model.AuthenticatorBridgeConnectionType
+import com.bitwarden.authenticatorbridge.provider.SymmetricKeyStorageProvider
+
+/**
+ * Factory for supplying implementation instances of Authenticator Bridge SDK interfaces.
+ */
+class AuthenticatorBridgeFactory(
```

Comment:
We should probably add a `@OmitFromCoverage` so we don't need to add unnecessary tests for this.

Reply:
Dizzy mentioned this as well. The tricky thing is that we don't have that annotation in this module... _yet_.

Reply:
Gotcha. Yeah the duplication of some things across modules (like some of the fakes too) is something we'll have to address as some point.